DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/IntitialMenuViewModelTests.cs

```diff
@@ -107,7 +107,7 @@ public void Initial_menu_banner_text_can_be_null()
         }
 
         [Test]
-        public void Initial_menu_include_certification_can_be_true()
+        public void Initial_menu_should_show_certification_summary_can_be_true()

```

Comment:
This test name isn't exactly accurate. We're showing completion summary not certification summary. And we're not testing that it can be true, we're testing that it's true if include certification is true 🙂